actioncable/lib/action_cable/subscription_adapter/redis.rb

```diff
@@ -0,0 +1,45 @@
+begin
+  require 'em-hiredis'
+  require 'redis'
+rescue Gem::LoadError => e
+  raise Gem::LoadError, "You are trying to use the Redis Action Cable adapter, but do not have the proper gems installed. Add `gem 'em-hiredis'` and `gem 'redis'` to your Gemfile (and ensure its version is at the minimum required by Action Cable)."
+end
+
+module ActionCable
+  module SubscriptionAdapter
+    class Redis < Base
```

Comment:
Wouldn't it be better to just mark these as `# :nodoc:` don't see anything here that users should be calling directly.